src/_guides/language/language-tour.md

```diff
@@ -125,10 +125,15 @@ mind:
 -   *Identifiers* can start with a letter or underscore (\_), followed by any
     combination of those characters plus digits.
 
--   Sometimes it matters whether something is an *expression* or a
-    *statement*, so it helps to be precise about those two words.
-
--   Dart tools can report two kinds of problems: warnings and errors.
+-   Dart has both *expressions* (which have runtime values) and
+    *statements* (which don't).
+    For example, the [conditional expression](#conditional-expressions)
+    `condition ? expr1 : expr2` has a value of `expr1` or `expr2`.
+    Compare that to an [if-else statement](#if-and-else), which has no value.
+    A statement often contains one or more expressions,
+    but an expression can't contain a statement.
```

Comment:
... can't directly contain ...

(otherwise pedants will say that `() { print(42); }` is an expression containing a statement).